src/measure_theory/integral/bochner.lean

```diff
@@ -777,7 +758,7 @@ begin
   { rw [integral_undef hf, norm_zero], exact to_real_nonneg }
 end
 
-lemma ennnorm_integral_le_lintegral_ennnorm (f : α → E) :
+lemma nnnorm_integral_le_lintegral_nnnorm (f : α → E) :
```

Comment:
I am not sure about this renaming: this lemma is about the coercion to ennreal of the nnnorm, i.e., the ennnorm.

Reply:
Oh, ok. I just saw nnnorm in the statement and thought this was a typo or an old name that we forgot to change. I'll undo the change.